row_cli.py

```diff
@@ -46,12 +48,25 @@ def main():
     args = docopt(__doc__, version="1.0")  # type: ignore
 
     if args["storage"] and args["generate-index"]:
-        index = row.generate_index(args["--from"], args["--save-to"])
+        if args["--prefix"]:
+            index = row.generate_index(args["--from"], args["--prefix"], args["--save-to"])
+        else:
+            index = row.generate_index(args["--from"], None, args["--save-to"])
+
```

Comment:
**suggestion**: this is another unnecessary if else. Pass the prefix no matter if it is present or not and you have the same outcome.